model/src/main/kotlin/CuratedPackage.kt

```diff
@@ -41,4 +41,18 @@ data class CuratedPackage(
      * A comparison function to sort packages by their identifier.
      */
     override fun compareTo(other: CuratedPackage) = pkg.id.compareTo(other.pkg.id)
+
+    /**
+     * Returns a [CuratedPackage] representing the same package as this one but which does not have any curations
+     * applied.
+     */
+    fun toUncuratedPackage(): CuratedPackage {
```

Comment:
This should rather return a `Package` object, because the list of curation results is always set to an empty list below.